Jamulus.pro

```diff
@@ -238,18 +238,53 @@ win32 {
     LIBS += -framework AVFoundation \
         -framework AudioToolbox
 } else:android {
-    ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+    # By default build for all the ABIs
+    # ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+
+    # get ANDROID_ABIS from environment - passed directly to qmake
+    ANDROID_ABIS = $$getenv(ANDROID_ABIS)
+
+    # Optional: if ANDROID_ABIS is passed as env var to qmake, will override this
+    # !defined(ANDROID_ABIS, var):ANDROID_ABIS = arm64-v8a
+
+    # Experiments show likely better results with Android 10+ devices
+    ANDROID_MIN_SDK_VERSION = 29
+    ANDROID_TARGET_SDK_VERSION = 32
     ANDROID_VERSION_NAME = $$VERSION
-    ANDROID_VERSION_CODE = $$system(git log --oneline | wc -l)
+
+    ## For local Dev use on Windows/WSA:
+    equals(QMAKE_HOST.os, Windows) {
+        ANDROID_ABIS = x86_64
+        ANDROID_VERSION_CODE = 1234 # dummy int value
+    } else {
+        # date-based unique integer value for Play Store submission
+        !defined(ANDROID_VERSION_CODE, var):ANDROID_VERSION_CODE = $$system(date +%s | cut -c 2-)
```

Comment:
See above - I'd rather remove this entirely and default to `1` and use the
```
 ANDROID_VERSION_CODE = $$num_add($$ANDROID_VERSION_CODE, ...)
```
approach.